### PR TITLE
Docs: Update-to-export-pattern

### PR DESCRIPTION
**Pull request by chloepoulterdesign**
I've made a couple of updates to the export pattern just to reflect the changes since it's being implemented. Nothing major

### CODE REVIEW

File: src/pages/experimental/export-pattern/index.mdx [hunk 1]
```diff
@@ -2,58 +2,124 @@
...
 
-![Example of an export pattern using a modal in context](images/contextual.png)
+![Example of an export modal in context](images/0.png)
+
+<Caption>Example of an export modal</Caption>
 
-<Caption>Example of a export pattern using a modal</Caption>
+![Example of a download modal in context](images/download0.png)
+
+<Caption>Example of a download modal in context</Caption>
```
Comment (connor-leech) [inline src/pages/experimental/export-pattern/index.mdx]:
```suggestion
<Caption>Example of a download modal in the context of a product</Caption>
```

File: src/pages/experimental/export-pattern/index.mdx [hunk 2]
```diff
@@ -2,58 +2,124 @@
...
 
 ## Intuitive default name
 
-When the resource is given an intuitive name by default and there is no choice of export location, the export occurs on the click of the Export button and nothing additional is displayed.
+When the resource is given an intuitive name by default and there is no choice of export/download location, the export occurs on the click of the Export button and nothing additional is displayed.
```
Comment (connor-leech) [inline src/pages/experimental/export-pattern/index.mdx]:
```suggestion
When a resource is given an intuitive name by default and there is no choice of an export or download location, the export occurs on the click of the Export button and nothing additional is displayed.
```

File: src/pages/experimental/export-pattern/index.mdx [hunk 3]
```diff
@@ -2,58 +2,124 @@
...
+
+![Example of a download button](images/download1.png)
+
+ </Column>
+</Row>
+
 ## Editable name
 
-When exporting with an editable name, a dialog should be presented to the user displaying an editable textbox that is pre-populated with the default name for the file.
+When exporting/downloading with an editable name, a dialog should be presented to the user displaying an editable textbox that is pre-populated with the default name for the file.
```
Comment (connor-leech) [inline src/pages/experimental/export-pattern/index.mdx]:
```suggestion
When a file's name can be edited before exporting or downloading, a dialog containing an editable textbox should be presented to the user. The textbox should be pre-populated with the file's default name.
```

File: src/pages/experimental/export-pattern/index.mdx [hunk 4]
```diff
@@ -2,58 +2,124 @@
...
 
 ![Example of an export button on a data table](images/1.png)
 
+</Column>
+</Row>
+
+<Row>
+ <Column colLg={8}>
+
+![Example of a download button](images/download1.png)
```
Comment (connor-leech) [inline src/pages/experimental/export-pattern/index.mdx]:
```suggestion
![Example of a download button](images/download1.png)

<Caption>Example of a download button</Caption>
```

File: src/pages/experimental/export-pattern/index.mdx [hunk 5]
```diff
@@ -2,58 +2,124 @@
...
-![Example of an export modal in its "Success" state.](images/3.5.png)
+<Caption>Example of an export modal in its "Exporting" state</Caption>
 
 </Column>
 </Row>
 
-## Specify export location
+## Optional extras
+
+### Specify export or download location
```
Comment (connor-leech) [inline src/pages/experimental/export-pattern/index.mdx]:
```suggestion
### Specify the export or download location
```

**Comment by chloepoulterdesign** [comment]
Hey @connor-leech I've committed your suggestions

**Comment by vpicone** [comment]
@chloepoulterdesign I went ahead and resolved the merge conflicts. If you don't mind, double check to make sure the images are correct and your changes were applied!

**Comment by chloepoulterdesign** [comment]
Thank you so much @vpicone it looks good to me!

**Comment by vpicone** [comment]
@chloepoulterdesign cool! @connor-leech you good?

**Pull request merged**